torch/quantization/fake_quantize.py

```diff
@@ -61,6 +97,10 @@ def __init__(self, observer=MovingAverageMinMaxObserver, quant_min=0, quant_max=
         self.qscheme = self.activation_post_process.qscheme
         self.ch_axis = self.activation_post_process.ch_axis \
             if hasattr(self.activation_post_process, 'ch_axis') else -1
+        assert _is_per_channel(self.qscheme) or \
+            _is_per_tensor(self.qscheme), \
+            'Only per channel and per tensor quantization are supported in fake quantize' + \
+            ' got qscheme: ' + str(self.qscheme)
 
     @torch.jit.export
```

Comment:
should the options to enable/disable fake_quant/observer go on the base class too?

Reply:
I think ideally these should be removed

Reply:
but as a middle step, it should live in base class as well.